app/models/event.rb

```diff
@@ -88,4 +88,21 @@ def calculate_gamedays
     end
     self.save
   end
+
+  def add_participant(user)
+    participants << user
+  end
+
+  def remove_participant(user)
+    participants.delete(user)
+  end
+
+  def has_participant?(user)
+    participants.include?(user)
+  end
+
+  # aliasing to ease renaming in database
+  def participants
+    users
```

Comment:
what does ease renaming in database mean? What and why should something be renamed in the database?
you should be able to name the association directly with I believe the class_name: option of the HABTM association